Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM sssd/sssd-deps
+
+MAINTAINER SSSD Maintainers <sssd-maint@redhat.com>
+
+ARG TARBALL
+
+RUN  echo -n | openssl s_client -connect scan.coverity.com:443 | sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' | sudo tee -a /etc/ssl/certs/ca- && curl -s https://scan.coverity.com/scripts/travisci_build_coverity_scan.sh -o /usr/bin/travisci_build_coverity_scan.sh && chmod a+x /usr/bin/travisci_build_coverity_scan.sh
```

Comment:
Did you craft this yourself or did you add this according to some documentation? If this is a "standard way" documented somewhere, it might be nice to add the URL.

Reply:
I've based SSSD work on libmodulemd work done by Stephen. So, this part is pretty much a c&p from his work.

Reply:
This is actually a copy-paste from the instructions on scan.coverity.com for setting up a Travis CI integration. As an admin, click on "submit build" and then the "Configure Travis CI" tab and you'll see it right there.